platform_db_backup.sh

```diff
@@ -81,6 +81,15 @@ if [[ -v AWS_BACKUP_BUCKET || -v SFTP_SERVER ]]; then
     mkdir -p "${DUMP_FOLDER}"
     echo "Creating database dump to ${DUMP_FILE}";
     time mysqldump --max-allowed-packet=16M --single-transaction --skip-opt -e --quick --skip-disable-keys --skip-add-locks -a --add-drop-table --triggers --routines -h "${DB_HOST}" -P "${DB_PORT}" -u "${DB_USER}" -p"${DB_PASS}" "${DB_NAME}" | gzip -9 > "${DUMP_FILE}"
+    if [[ -n $SECRET_ENC_PASS ]] && [[ "$ENABLE_ENCRYPTION" == "1" ]]; then
```

Comment:
I think we'd also need some logging when the secret is missing and encryption was unsuccessful. 
I'd add else statement with the echo "FAILED, because ..."

Reply:
logging added

Reply:
I'm a logging freaq.